msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -459,4 +479,26 @@ Specify your AAD App ID and Graph information to help users seamlessly sign into
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
 |`id`|String|36 characters|✔|AAD application id of the app. This id must be a GUID.|
-|`resource`|String|2048 characters|✔|Resource url of app for acquiring auth token for SSO.|
\ No newline at end of file
+|`resource`|String|2048 characters|✔|Resource url of app for acquiring auth token for SSO.|
+
+## configurableProperties
+
+**Required**
+
+The `configurableProperties` block defines the app properties that an admin can customize. The developer must define them in the manifest.
+
+> [!NOTE]
+> A minimum of one property must be defined. The developer can define a maximum of nine properties in this block.
+
+|Name| Type| Maximum size | Required | Description|
+|----|-----|--------------|----------|------------|
+|`type`|array|            |✔|                     |
+|`description`|String|128 characters|✔|Description of the list of properties.|
+|`maxItems`| array of enum|9|✔| List of configurable properties.|
+|`items`| | |✔| |
+
+> [!NOTE]
```

Comment:
could you also list the enum string values here similar how its done for "devicePermissions" section above in this file

Reply:
Hi @vganiga - I have added the list of enum string values. Please validate if the table information is correct and also provide inputs for `type` in the table. Sharing a snapshot for reference.
![image](https://user-images.githubusercontent.com/70375155/112596805-50339980-8e32-11eb-9729-a0eba3608d74.png)
Thank you.